vignettes/Example2-Ostats-multivariate.Rmd

```diff
@@ -22,19 +22,21 @@ library(Ostats)
 data(pitcher_traits)
 ```
 
+# Example 2: Multivariate Example - Pitcher plant leaf traits across populations
+
 # Overview
 
-This vignette on multi-variable community-level trait overlap statistics is intended for users who are already familiar with the single-variable O-statistics we introduce in the [first package vignette](Ostats-introduction.html).
+This vignette on multivariate trait overlap statistics is intended for users who are already familiar with the univariate O-statistics we introduce in the [first package vignette](Ostats-univariate.html).
```

Comment:
See above comment, I think it is weird to delete all reference to community. Yes of course you "could" do this on data that is not an ecological community, but why are we getting rid of all mention of the word? It is still, to me, by far the most relevant use for this stuff

Reply:
added community back in